doc/source/misc.rst

```diff
@@ -72,6 +73,55 @@ Further reading:
 - https://stackoverflow.com/questions/44379525/r-reticulate-how-do-i-clear-a-python-object-from-memory
 
 
+Keeping a fork in sync with master
```

Comment:
I agree with @Trybnetic 's comment about this: This is a duplicate of github's [docs](https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/syncing-a-fork). 
We simply could link to this in the developer.rst

Reply:
I still feel that I want to have it in for now. It still gives you a good indication what to do without clicking on a link and documents how (I / we) do it.